titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/topology/DefaultTopologyFactory.java

```diff
@@ -146,15 +149,20 @@ private static int getJobMaxSkew(Job<?> job) {
         }
 
         DisruptionBudgetPolicy policy = job.getJobDescriptor().getDisruptionBudget().getDisruptionBudgetPolicy();
+        // Job spreading is only relevant for jobs that care about the availability.
         if (!(policy instanceof AvailabilityPercentageLimitDisruptionBudgetPolicy)) {
-            return 1;
+            return -1;
         }
         int jobSize = getJobDesiredSize(job);
         if (jobSize <= 1) {
             return 1;
```

Comment:
Why not just disable the topology spread by returning -1?

Reply:
Good point. Will do that.